src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1177,6 +1181,28 @@ private void SetRequestHandleBufferingOptions(SafeWinHttpHandle requestHandle)
             SetWinHttpOption(requestHandle, Interop.WinHttp.WINHTTP_OPTION_MAX_RESPONSE_DRAIN_SIZE, ref optionData);
         }
 
+        private void SetRequestHandleHttp2Options(SafeWinHttpHandle requestHandle, Version requestVersion)
+        {
+            Debug.Assert(requestHandle != null);
+#if NET46
+            if (requestVersion == HttpVersion20)
+#else
+            if (requestVersion == HttpVersion.Version20)
+#endif   
```

Comment:
If we do need to use the compilation constant, could we at least consolidate them all to a single location, e.g. have a property that has a getter which uses the constant, and then all call sites use that property rather than having each call site have an if/else like this?